src/context-parser-handlebars.js

```diff
@@ -894,6 +963,25 @@ ContextParserHandlebars.prototype.handleRawBlock = function(input, i, saveToBuff
     throw "Parsing error! Cannot encounter '}}}}' close brace of raw block.";
 };
 
+/**
+* @function ContextParserHandlebars.reset
+*
+* @description
+* All non-config internal variables are needed to reset!
+*/
+ContextParserHandlebars.prototype.reset = function() {
```

Comment:
I finally find it at https://github.com/yahoo/secure-handlebars/pull/120/files#diff-7744afc9654785f847f9d3e31548ad70R60